security/pkg/server/ca/monitoring.go

```diff
@@ -67,8 +66,7 @@ var (
 	)
 	certChainExpiryTimestamp = monitoring.NewGauge(
 		"citadel_server_cert_chain_expiry_timestamp",
-		"The unix timestamp, in seconds, when Citadel cert chain will expire. "+
-			"A negative time indicates the cert is expired.",
+		"The unix timestamp, in seconds, when Citadel cert chain will expire. ",
```

Comment:
I wonder what it should return in terms of error since negative values are valid timestamps (at least in Go)